### PR TITLE
Add 3 icons

### PR DESCRIPTION
**Pull request by SiMah5**
## Description

added icons for SimpleX , ArtStation and Microsoft Edge Dev apps 

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* SimpleX (`chat.simplex.app`) - added from icon request form
* Microsoft Edge Dev (`com.microsoft.emmx.dev`) - added from icon request form
* ArtStation (`com.ballistiq.artstation`)

### CODE REVIEW

File: app/assets/appfilter.xml [hunk 1]
```diff
@@ -210,6 +210,7 @@
   <item component="ComponentInfo{com.donnnno.arcticons/com.donnnno.arcticons.activities.SplashActivity}" drawable="arcticons_dark" name="Arcticons Dark" />
   <item component="ComponentInfo{de.swr.avp.ard/de.ard.ardmediathek.ui.main.MainActivity}" drawable="ard_mediathek" name="ARD Mediathek" />
   <item component="ComponentInfo{de.swr.avp.ard/de.swr.avp.ard.MainActivity}" drawable="ard_mediathek" name="ARD Mediathek" />
+  <item component="ComponentInfo{com.ballistiq.artstation/com.ballistiq.artstation.view.login.LoginActivity}" drawable="artstation" name="ArtStation"/>
```
Comment (Chefski) [inline app/assets/appfilter.xml:213]:
```suggestion
  <item component="ComponentInfo{com.ballistiq.artstation/com.ballistiq.artstation.view.login.LoginActivity}" drawable="artstation" name="ArtStation" /> 
```

File: app/assets/appfilter.xml [hunk 2]
```diff
@@ -3746,6 +3748,7 @@
   <item component="ComponentInfo{io.simplelogin.android/io.simplelogin.android.module.startup.StartUpActivity}" drawable="simple_login" name="SimpleLogin" />
   <item component="ComponentInfo{com.automattic.simplenote/com.automattic.simplenote.NotesActivity}" drawable="simplenote" name="Simplenote" />
   <item component="ComponentInfo{felixwiemuth.simplereminder/felixwiemuth.simplereminder.ui.reminderslist.RemindersListActivity}" drawable="simple_reminder" name="SimpleReminder" />
+  <item component="ComponentInfo{chat.simplex.app/chat.simplex.app.MainActivity_default}" drawable="simplex" name="SimpleX"/>
```
Comment (Chefski) [inline app/assets/appfilter.xml:3751]:
```suggestion
  <item component="ComponentInfo{chat.simplex.app/chat.simplex.app.MainActivity_default}" drawable="simplex" name="SimpleX" /> 
```

File: svgs/artstation.svg
Comment (Chefski) [inline svgs/artstation.svg:1]: The stroke is slightly over 12px. The icon is also slightly off center. Could you fix both issues?

File: svgs/simplex.svg
Comment (Chefski) [inline svgs/simplex.svg:1]: Could you round the endpoints on the lines and ensure the icon fits the 160x160px content area.

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/simplex.svg
Comment (WHOMAN5) [reply svgs/simplex.svg:1]: yeah ofc

**Comment by SiMah5** [comment]
uh, ok so i did resize them in figma (i also re-checked their sizes in inkscape by creating a 160x160 doc and dropping them inside it) and they both look to be the correct size on my end. i dont know what im doing wrong tbh.
image

can you tell me what software and method you're using for measuring the icons?

**Comment by Chefski** [comment]
I just have guidelines for 160x160 and 144x144. If you group both paths, is the width 148?

image

**Comment by SiMah5** [comment]
oh im so dumb, i completely misunderstood you when you said i should fit them into the 160x160 area. 
gonna fix them now, sorry.

**Review by Chefski — APPROVED** [review]

**Comment by Chefski** [comment]
Yeah I don't like the word "fit" to use as making it just get to the edges. Its a little confusing. Thanks for making the changes though

**Pull request merged**